src/snooping.jl

```diff
@@ -1,23 +1,46 @@
 using Pkg, Serialization
 
 
-function snoop(package, tomlpath, snoopfile, outputfile, reuse = false, blacklist = [])
```

Comment:
I'm using this function. What is the motivation for removing it? What will be the upgrade path?

Reply:
Also addressed in https://github.com/JuliaLang/PackageCompiler.jl/pull/204/commits/e12e89951919df8a82a235143ea4b2a17b9e4eb5 as far as I can tell.